packages/@vuepress/markdown/__tests__/plugins/assetsPlugin.spec.ts

```diff
@@ -189,6 +190,41 @@ describe('@vuepress/markdown > plugins > assetsPlugin', () => {
       '<img src=".../汉字.png">',
       '<img src=".../100%.png">',
       '<img alt="attrs" src=".../attrs.png" width="100px">',
+
+      /* srcset */
+
+      // relative paths
+      '<img srcset="./foo.png 1x, ../sub/foo.png 2x, ./foo/bar.png 1024w ,../sub/foo/bar.png 2048w, ../baz.png 4096w,../../out.png">',
+      '<img srcset="./汉字.png 1x, ./100%.png">',
+      '<img alt="attrs" srcset="./attrs.png" width="100px">',
+      // aliases
+      '<img srcset="@alias/foo.png 1x,@alias/汉字.png 2x, @alias/100%.png 3x">',
```

Comment:
Missing whitespace after the first comma.

It could be better to add extra cases without the comma, e.g.:

```ts
// no whitespace after comma
'<img .....'
```

Reply:
That's expected, because that unit test is to mixed space position and see if it works. I am not sure whether it's necessary to add a lot of combination into the unit test.